src/account/sync/input_selection.rs

```diff
@@ -156,9 +206,8 @@ mod tests {
                 .build()
                 .unwrap();
             available_utxos.push(super::Input {
-                address: address.address().clone(),
-                balance: address.balance(),
                 internal: false,
+                output: address.outputs().iter().next().expect("No output").1.clone(),
```

Comment:
Maybe it would be nice to have a comment for this line.

Reply:
Will refactor the whole section